llvm/lib/Target/EraVM/EraVMInstrFormats.td

```diff
@@ -231,6 +231,7 @@ def COND_E    : EraVMCondCode<3>;
 def COND_GE   : EraVMCondCode<4>;
 def COND_LE   : EraVMCondCode<5>;
 def COND_NE   : EraVMCondCode<6>;
+def COND_OF   : EraVMCondCode<8>;
```

Comment:
Please document here that it's a pseudo flag, alias of LT and that we added it to distinguish between `lt` set by `sub` instruction which is reversible and `lt` set to indicate overflow by `add`, `mul` which aren't.

Reply:
Will add comments.